ledger/participant-state/kvutils/tools/src/test/scala/ledger/participant/state/kvutils/tools/integritycheck/StateUpdateComparisonSpec.scala

```diff
@@ -31,7 +31,7 @@ final class StateUpdateComparisonSpec
       val right = aConfigurationChangeRejected.copy(rejectionReason = "another reason")
 
       ReadServiceStateUpdateComparison
-        .compareUpdates(left, right, List.empty, List.empty)
+        .compareUpdates(left, right, List.empty, List.empty, List.empty)
```

Comment:
```suggestion
        .compareUpdates(left, right, Iterable.empty, Iterable.empty, Iterable.empty)
```

for coherence with the usage below